int32 count
float32[] data
